lab-jaren/test/read-many-files-test.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
```

Comment:
Great job going above and beyond attempting the bonus AND writing tests for it. I don't have an answer to solving the async issue yet, but definitely go to Duncan if you're interested in solving it.